.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 dist: trusty
 language: node_js
-node_js: 8.11
+node_js: node
 
 cache:
```

Comment:
this might cause issues, see https://github.com/kenwheeler/cash/pull/233 for details

Reply:
Removed.